docs/content/getting-started.md

```diff
@@ -88,22 +117,27 @@ type Todo struct {
 }
 ```
 
-And then tell gqlgen to use this new struct by adding this to the gqlgen.yml:
+Next tell gqlgen to use this new struct by adding it to `gqlgen.yml`:
+
 ```yaml
 models:
   Todo:
     model: github.com/vektah/gqlgen-tutorials/gettingstarted.Todo
```

Comment:
Earlier the doc says:

> `mkdir -p $GOPATH/src/github.com/[username]/gqlgen-todos`

So I think this doesn't match. Probably should be:

`model: github.com/[username]/gqlgen-todos.Todo`